examples/mo_hpo/mo_hyperband.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
I am not sure if we are going to include the notebooks directly in example folder, since it's impossible to review and diff. One way to workaround is using markdown, but it requires a little bit extra steps for user to run.

Reply:
I am open to converting the notebook to markdown if you think this is the better way. Please, let me know how you would like to proceed with the example notebook.

Reply:
Could you convert it to markdown, so we do the same as everywhere else? This also serves as additional tests then